spec/features/publisher_information/visit_publisher_updates_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+
+RSpec.feature "User is on the home page and can visit the publisher updates page" do
+  scenario do
+    given_im_on_the_home_page
+    and_click_on_the_whats_new_page_link
+    then_i_can_see_publisher_update_page
+  end
+
+  def given_im_on_the_home_page
+    visit "/"
+  end
+
+  def and_click_on_the_whats_new_page_link
+    click_on "Publisher updates"
+  end
+
+  def then_i_can_see_publisher_update_page
+    expect(page).to have_content("Summary of updates")
```

Comment:
We should probably check this content matches the translation key so it's not coupled to the text e.g: https://github.com/alphagov/content-publisher/blob/7ffd2da095035dbb176fd41429185e180fbe3b50/spec/features/editing_topics/edit_topics_publishing_api_down_spec.rb#L29

Reply:
Done.  It now uses the summary key in the main publisher updates yml file.